App.tsx

```diff
@@ -88,14 +105,19 @@ const defaultDate = new Date();
 
 const App = () => {
   try {
+    // we need to subscribe to the react-query cache in order to remove
+    // images from the local filesystem when their TTL expires
+    React.useEffect(() => {
+      return queryCache.subscribe(event => ImageCache.cleanup(event));
+    }, []);
```

Comment:
whoa this is a spooky level of indirection. I only realized this connection existed on my second pass through the PR. 

suggestion: make a method in the ImageCache called `initialize` that contains this code, and pass the `queryCache` into it. 
```suggestion
    React.useEffect(() => {
      return ImageCache.initialize(queryCache);
    }, []);
```

Reply:
Oops, this is before I realized that variables in the global scope outside of the app can't change, so we don't need `useEffect` here - let me hoist this up to the top of the file